test/parallel/parallel.status

```diff
@@ -9,6 +9,9 @@ prefix parallel
 [$system==win32]
 
 [$system==linux]
+# https://github.com/nodejs/node/issues/22308
+test-cluster-master-kill: PASS,FLAKY
+test-cluster-master-error: PASS,FLAKY
```

Comment:
The host/platform that these were failing on has been removed, so I think there's no need to mark them as flaky. (For what it's worth, they were failing 100% of the time and I'm not a fan of marking 100% failures as flaky.)

Reply:
On the other hand, I'd like to see Alpine 3.8 back in rotation. We're risking either ignoring Alpine 3.8 completely, or if we mark this as flaky we risk not fixing the underlying problem. Pick your poison.

Reply:
I'm also against adding this, since it'll create a permanent yellow CI, and that is desensitizing...
We have a tracking issue, and we have eyes on the problem, so the chances of loosing track are lower.